lib/core/viewer/components/imgViewer.component.scss

```diff
@@ -22,7 +24,7 @@
 
             .adf-toolbar .mat-toolbar {
                 max-height: 48px;
-                background-color: mat-color($primary, default-contrast, 1);
+                background-color: mat-color($background, card, 0.87);
```

Comment:
Is that the color that corresponds to the design spec?

Reply:
Why Viewer needs to user the styles of Card?

Reply:
Do we have design spec for the dark theme?

Reply:
It doesn't need to use it, but this is the default panel background color in the material theming:

```js
$mat-dark-theme-background: (
  app-bar:    map_get($mat-grey, 900),
  background: #303030,
  card:       map_get($mat-grey, 800),
  dialog:     map_get($mat-grey, 800),
);
```

We don't have anything general for "panels", at least I don't know anything like that, except the card.